app/package.json

```diff
@@ -14,6 +14,8 @@
     "@babel/core": "^7.13.10",
     "@react-three/drei": "^4.3.3",
     "@react-three/fiber": "^6.0.16",
+    "@tanstack/react-query": "^4.20.4",
+    "@tanstack/react-query-devtools": "^4.20.4",
```

Comment:
I assume this should be a dev dependency instead?

Reply:
Hmm, in the documentation of react-query, they don't specify that you should install them as a dev dependency: https://tanstack.com/query/v4/docs/react/devtools#install-and-import-the-devtools

Not sure about this one.

Reply:
> By default, React Query Devtools are only included in bundles when process.env.NODE_ENV === 'development', so you don't need to worry about excluding them during a production build.

I see. Not a big fan of that but I guess we can leave them in the prod build then.